python/hopsworks_common/constants.py

```diff
@@ -191,7 +191,6 @@ class RESOURCES:
     MIN_GPUS = 0
```

Comment:
I would just call it `GPUS`, since `MIN_GPUS` implies there can be a `MAX_GPUS` but there's not.